apps/helptheducks.com/src/app/page.tsx

```diff
@@ -64,7 +64,7 @@ const Hero = () => {
   );
 };
 
-const PetitionSection = () => {
+function PetitionSection() {
```

Comment:
is there some new eslint rule we have enabled that's complaining about this, or just your preference?

Reply:
just preference. but after looking into it i realized it's not objectively better to use named functions in place of arrow function variables, but it's just less necessary to use arrow functions now than before. but i do think it's a little better readability-wise. what do you think?